telegram.html

```diff
@@ -0,0 +1,37 @@
+<!doctype html>
+<html lang='en'>
+
+<head>
+    <title>Handshake Telegram</title>
+
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
+    <meta http-equiv='Content-Type' content='text/html; charset=utf-8' />
+    <meta name="description" content="Decentralized certificate authority and naming" />
+
+    <!-- Facebook Meta Tags -->
+    <meta property="og:url" content="https://handshake.org/telegram">
+    <meta property="og:type" content="website">
+    <meta property="og:title" content="Handshake Telegram Group">
+    <meta property="og:description" content="Decentralized certificate authority and naming">
+    <meta property="og:image" content="https://www.handshake.org/images/landing/logo-dark.svg">
+
+    <!-- Twitter Meta Tags -->
+    <meta name="twitter:card" content="summary_large_image">
+    <meta property="twitter:domain" content="handshake.au">
+    <meta property="twitter:url" content="https://handshake.org/telegram">
+    <meta name="twitter:title" content="Handshake Telegram Group">
+    <meta name="twitter:description" content="Decentralized certificate authority and naming">
+    <meta name="twitter:image" content="https://www.handshake.org/images/landing/logo-dark.svg">
+
+    <link rel="shortcut icon" href="https://handshake.org/img/favicon/hns-favicon.ico" type="image/x-icon">
+
+    <!-- After 0 seconds redirect user to Discord Invite Link -->
```

Comment:
```suggestion
    <!-- After 0 seconds redirect user to telegram channel -->
```

Reply:
🤦‍♂️That's what you get when you copy paste code. I'll update that in a bit

Reply:
This is fixed in 41cd16c0edc4f75169a41096c52a42a9d9c781d6